examples/SC_noise_capture.py

```diff
@@ -0,0 +1,148 @@
+"""
+SpinCore Noise Acquisition
+==========================
+
+This script acquires a set number of captures of the noise
+as seen by the SpinCore. Remember to add a terminated attenuator 
+to the output of the SpinCore prior to measurement!
+"""
+import pyspecdata as ps
+import numpy as np
+import os
+from numpy import r_
+from timeit import default_timer as timer
```

Comment:
used for start and end times to print total acquisition time at end (collect100.py also does this)